app/components/publications/documents/document-card-step.js

```diff
@@ -1,7 +1,44 @@
 import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
 import { task } from 'ember-concurrency';
 
+/**
+ * @argument {Piece} piece
+ * @argument {boolean} isEnabledDelete
+ * @argument {boolean} isViaCouncilOfMinisters
+ * @argument {Function} onDelete
+ */
 export default class PublicationsDocumentsDocumentCardStepComponent extends Component {
+  @tracked isOpenWordDocumentUploadModal;
+
+  get hasWordVersionOfDocument() {
+    const file = this.args.piece.file;
```

Comment:
Await this relation in a task, and don't show the layout until the task has been finished (so this getter always has `this.args.piece.file`)